pandas/core/generic.py

```diff
@@ -1078,39 +1043,50 @@ def _rename(
     def rename_axis(
         self: NDFrameT,
         mapper: IndexLabel | lib.NoDefault = ...,
+        index=...,
+        columns=...,
+        axis: Axis = ...,
+        copy: bool_t = ...,
```

Comment:
can these be keyword-only?

Reply:
We could deprecate and make them keyword only in a follow up, but as is they cannot be yet.

Reply:
I'm confused - why can't they be made keyword-only?

These arguments (e.g. `index`) already can't be passed positionally, so which behaviour which currently works would break?

Reply:
Ah shoot I misread the previous signature. Yeah these should be able to be keyword only